### PR TITLE
Adding MIRACL 2CR for mdpr-tied-pft-msmarco-ft-miracl

### PR DESCRIPTION
**Pull request by jasper-xian**
Adding 2CR results from finetuning [castorini/mdpr-tied-pft-msmarco](https://huggingface.co/castorini/mdpr-tied-pft-msmarco) in-language with MIRACL.

### CODE REVIEW

File: pyserini/prebuilt_index_info.py [hunk 1]
```diff
@@ -6884,4 +6884,16 @@
         "downloaded": False,
         "texts": "wiki-all-6-3-tamber"
     },
+    "miracl-v1.0-sw-mdpr-tied-pft-msmarco-ft-miracl-sw": {
+        "description": "Faiss index for MIRACL v1.0 (Swahili) corpus encoded by mDPR passage encoder pre-fine-tuned on MS MARCO, then fine-tuned in-language with MIRACL.",
+        "filename": "miracl-v1.0-sw-mdpr-tied-pft-msmarco-ft-miracl-sw.tar.gz",
```
Comment (jasper-xian) [inline pyserini/prebuilt_index_info.py:6889]: I left these filenames here for now, but they don't seem to match the format of other index filenames. Happy to change if needed.

File: pyserini/prebuilt_index_info.py [hunk 2]
```diff
@@ -7263,4 +7263,212 @@
         "downloaded": False,
         "texts": "wiki-all-6-3-tamber"
     },
+    "miracl-v1.0-ar-mdpr-tied-pft-msmarco-ft-miracl-ar": {
+        "description": "Faiss index for MIRACL v1.0 (Arabic) corpus encoded by mDPR passage encoder pre-fine-tuned on MS MARCO, then fine-tuned in-language with MIRACL.",
+        "filename": "faiss.miracl-v1.0-ar.mdpr-tied-pft-msmarco-ft-miracl-ar.20230329.e40d4a.tar.gz",
+        "readme": "faiss.miracl-v1.0-ar-mdpr-tied-pft-msmarco-ft-miracl-ar.20230329.e40d4a.README.md",
```
Comment (lintool) [inline pyserini/prebuilt_index_info.py:7269]: We're missing a README? We can have a single README shared by all the languages...

**Review by lintool — CHANGES_REQUESTED** [review]
Missing README?

File: pyserini/prebuilt_index_info.py (hunk 2 above)
Comment (jasper-xian) [reply pyserini/prebuilt_index_info.py:7269]: Yeah we do have one, I'll correct the README name

File: pyserini/resources/index-metadata/faiss.miracl-v1.0.mdpr-tied-pft-msmarco-ft-miracl.20230329.e40d4a.README.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# miracl-v1.0-mdpr-tied-pft-msmarco-ft-all
```
Comment (lintool) [inline pyserini/resources/index-metadata/faiss.miracl-v1.0.mdpr-tied-pft-msmarco-ft-miracl.20230329.e40d4a.README.md:1]: I think `ft-all` is a typo here?

**Review by lintool — APPROVED** [review]

**Pull request merged**